otherlibs/threads/stdlib.ml

```diff
@@ -634,3 +635,53 @@ let exit retcode =
   sys_exit retcode
 
 let _ = register_named_value "Pervasives.do_at_exit" do_at_exit
+end
+
+include Pervasives
+
+(*MODULE_ALIASES*)
+module Arg          = Arg
+module Array        = Array
+module ArrayLabels  = ArrayLabels
+module Buffer       = Buffer
+module Bytes        = Bytes
+module BytesLabels  = BytesLabels
+module Callback     = Callback
+module Char         = Char
+module Complex      = Complex
+module Digest       = Digest
+module Ephemeron    = Ephemeron
+module Filename     = Filename
+module Format       = Format
+module Gc           = Gc
+module Genlex       = Genlex
+module Hashtbl      = Hashtbl
+module Int32        = Int32
+module Int64        = Int64
+module Lazy         = Lazy
+module Lexing       = Lexing
+module List         = List
+module ListLabels   = ListLabels
+module Map          = Map
+module Marshal      = Marshal
+module MoreLabels   = MoreLabels
+module Nativeint    = Nativeint
+module Obj          = Obj
+module Oo           = Oo
+module Parsing      = Parsing
+module Printexc     = Printexc
+module Printf       = Printf
+module Queue        = Queue
+module Random       = Random
+module Scanf        = Scanf
+module Set          = Set
+module Sort         = Sort
+module Spacetime    = Spacetime
+module Stack        = Stack
+module StdLabels    = StdLabels
+module Stream       = Stream
+module String       = String
+module StringLabels = StringLabels
+module Sys          = Sys
+module Uchar        = Uchar
+module Weak         = Weak
```

Comment:
Indeed. I think the right place for this list is stdlib.ml/mli, since I expect that most people will only edit .ml/.mli files and don't really know how the Makefiles work. Plus it's more wysiwyg. The duplication between the .ml and the .mli can't really be avoided, but the compiler will complain if they go out of sync.

I'll look at genenerating the Makefile lists at the same time as replacing the awk scripts.